app/src/main/java/org/oppia/android/app/player/state/itemviewmodel/DragAndDropSortInteractionViewModel.kt

```diff
@@ -129,6 +131,10 @@ class DragAndDropSortInteractionViewModel private constructor(
       this@DragAndDropSortInteractionViewModel.writtenTranslationContext
   }.build()
 
+  override fun getRawUserAnswer(): RawUserAnswer {
+    return RawUserAnswer.getDefaultInstance()
```

Comment:
Is this correct? We should be retaining the state for this interaction, too. Ditto for the selection interaction.

Reply:
I have tried to retain state in other interactions but it seems quite difficult and can you look at my current implementation. Thanks

Reply:
Updated for selectionInteraction